### ISSUE TITLE
Press style not removed when using disabled prop

### ISSUE THREAD

**Issue by ioetbc** [tamagui/tamagui#2445]
### Current Behavior

I have a disabled prop attached to a button that's controlled with state.
I also have a pressStyle attached to the same Tamagui button
When I click the buttton the pressStyle is never released

The pressStyle works perfectly if I remove the disabled prop. However, with the disabled prop when I click the button the pressStyle is not removed.

<img width="386" alt="image" src="https://github.com/tamagui/tamagui/assets/24758676/b942722f-6875-4c6c-985f-a83f4281060a">

**Once I disable the orange button and click it the pressStyle persists and is not removed. Now I am stuck with a yellow button**
<img width="377" alt="image" src="https://github.com/tamagui/tamagui/assets/24758676/059d5c18-9989-4807-94fd-8593c79d4568">

### Expected Behavior

PressStyle is released.

### Tamagui Version

```markdown
1.91.4
```

### Platform (Web, iOS, Android)

```markdown
IOS
```

### Reproduction

```markdown
https://github.com/ioetbc/tamagui-issue/blob/main/packages/app/features/home/screen.tsx
```

### System Info

```markdown
System:
    OS: macOS 13.4
    CPU: (10) arm64 Apple M1 Pro
    Memory: 85.38 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.11.1 - ~/.nvm/versions/node/v20.11.1/bin/node
    Yarn: 4.1.0 - /usr/local/bin/yarn
    npm: 10.2.4 - ~/.nvm/versions/node/v20.11.1/bin/npm
    Watchman: 2024.01.22.00 - /opt/homebrew/bin/watchman
  Browsers:
    Chrome: 122.0.6261.129
    Firefox: 109.0
    Safari: 16.5
```

**Comment by TheEhsanSarshar** [comment]
just tested and it's working fine on ios
try with latest version of tamagui

**Comment by xuanmai-agilityio** [comment]
Hi @TheEhsanSarshar I still have this issue on the latest Tamagui version, could you specify which version you're on?

**Comment by TheEhsanSarshar** [comment]
tested on the Latest version!

**Comment by xuanmai-agilityio** [comment]
@TheEhsanSarshar  It does not work with the latest version as I followed the example of @ioetbc . Could you take a look at the video below?

https://github.com/tamagui/tamagui/assets/61130765/8f2ba4fe-4248-4c50-a782-5f2ad197606d

But I realize when the initial `disabled` value is `true`, then we toggle it disabled => enabled. It works well after that.

**Comment by Nasseratic** [comment]
Same issue here

**Comment by Nasseratic** [comment]
I am experiencing the same issue as well

**Comment by xuanmai-agilityio** [comment]
@Nasseratic  the issue has been fixed on the version v1.95.1.
https://github.com/tamagui/tamagui/releases/tag/v1.95.1
